web/src/test/java/org/openmrs/module/filter/ModuleFilterDefinitionTest.java

```diff
@@ -12,24 +12,22 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThrows;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import javax.xml.parsers.DocumentBuilder;
 import javax.xml.parsers.DocumentBuilderFactory;
 import java.io.StringReader;
 import java.util.List;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 import org.openmrs.module.Module;
 import org.openmrs.module.ModuleException;
 import org.openmrs.module.web.filter.ModuleFilterDefinition;
-import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.openmrs.test.jupiter.BaseContextSensitiveTest;
 import org.w3c.dom.Document;
 import org.xml.sax.InputSource;
 
-@PrepareForTest(ModuleFilterDefinition.class)
-public class ModuleFilterDefinitionTest {
+public class ModuleFilterDefinitionTest extends BaseContextSensitiveTest {
```

Comment:
try removing the extends. I think this ModuleFilterDefinition can easily be unit tested. no mocks :)